inference-engine/src/vpu/myriad_plugin/CMakeLists.txt

```diff
@@ -42,7 +42,12 @@ target_link_libraries(${TARGET_NAME}
         mvnc inference_engine_legacy vpu_graph_transformer)
 
 # MyriadPlugin is not safe to unload it at runtime
-if(LINUX AND LINUX_OS_NAME MATCHES "Ubuntu")
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "^(arm.|ARM.)")
+    set(ARM ON)
+endif()
```

Comment:
`ARM` is already defined, no needs to do it once again

Reply:
Got it, thank you for the note